images/README.md

```diff
@@ -1 +1,11 @@
-This repo contains all windows equivalent of docker images used to run k8s e2e tests.
\ No newline at end of file
+This repo contains all windows equivalent of docker images used to run k8s e2e tests.
+
+## Image Repository List
```

Comment:
Can we link to this from: https://github.com/kubernetes-sigs/windows-testing#method-1-using-an-existing-cluster